toolchain/language_server/testdata/basics/fail_shutdown_without_exit.carbon

```diff
@@ -0,0 +1,39 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// AUTOUPDATE
+// TIP: To test this file alone, run:
+// TIP:   bazel test //toolchain/testing:file_test --test_arg=--file_tests=toolchain/language_server/testdata/basics/fail_shutdown_without_exit.carbon
+// TIP: To dump output, run:
+// TIP:   bazel run //toolchain/testing:file_test -- --dump_output --file_tests=toolchain/language_server/testdata/basics/fail_shutdown_without_exit.carbon
+
+// --- STDIN
+
+[[@LSP-CALL:initialize]]
+[[@LSP-CALL:shutdown]]
+
+// --- AUTOUPDATE-SPLIT
+
+// CHECK:STDERR: I[20:47:00.642] Warning: Missing Content-Length header, or zero-length message.
+// CHECK:STDERR: I[20:47:00.643] Warning: Missing Content-Length header, or zero-length message.
```

Comment:
FYI, these occur due to the extra whitespace in the file, which slightly shifts the result. I'm going to commit a fix since this is subtle and pretty mechanical, and merge.

Reply:
And also committing because it interferes with tests, to be sure (due to the timestamp).

See:
https://github.com/carbon-language/carbon-lang/pull/4916/commits/f7dfb59ef94f37e38a1c945064dfc5d871425c3b